YoCodeAutomatedTests/testappsettings.json

```diff
@@ -0,0 +1,8 @@
+﻿{
+  "AutomatedTesting": {
+    "TestDataPath": "C:\\YoCodeTestData"
+  },
+  "YoCodeLocation": {
+    "DLLFolderPath":  "C:\\Users\\ukmaug\\source\\repos\\YoCode\\YoCode\\bin\\Debug\\netcoreapp2.1"
+  }
+}
```

Comment:
This file should not be in git because it will be different for different people.